src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -211,7 +211,7 @@ void WindowMoveHandlerPrivate::MoveSizeUpdate(HMONITOR monitor, POINT const& ptS
 
 void WindowMoveHandlerPrivate::MoveSizeEnd(HWND window, POINT const& ptScreen, const std::map<HMONITOR, winrt::com_ptr<IZoneWindow>>& zoneWindowMap) noexcept
 {
-    if (window != m_windowMoveSize && !IsInterestingWindow(window, m_settings->GetSettings()->excludedAppsArray))
+    if (window != m_windowMoveSize || !IsInterestingWindow(window, m_settings->GetSettings()->excludedAppsArray))

```

Comment:
I think it was correct before.

Reply:
Oh, double checked. It was 
```
if (window == m_windowMoveSize || IsInterestingWindow(window)) 
{
...
}
```
so by De Morgan's law it will be
```
if (window != m_windowMoveSize && !IsInterestingWindow(window)) 
{
    return;
}
```